pkg/reconciler/kubernetes/tektonhub/tektonhub.go

```diff
@@ -139,18 +139,30 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, th *v1alpha1.TektonHub)
 
 	// Manage DB
 	if err := r.manageDbComponent(ctx, th, hubDir, version); err != nil {
+		if err == v1alpha1.RECONCILE_AGAIN_ERR {
+			r.enqueueAfter(th, 10*time.Second)
+			return nil
+		}
 		return err
 	}
```

Comment:
```suggestion
	if err := r.manageDbComponent(ctx, th, hubDir, version); err != nil {
		return r.handleError(err)
	}
```

define a function
```
func (r *reconciler) handleError(err error) error {
    if err == v1alpha1.RECONCILE_AGAIN_ERR {
			r.enqueueAfter(th, 10*time.Second)
			return nil
		}
		return err
}
```

Reply:
Yeah i did that, just wanted to confirm if the logic behind the errors was correct 👍🏻 
will update the patch